src/components/context_menu/context_menu_panel.tsx

```diff
@@ -270,7 +270,10 @@ export class OuiContextMenuPanel extends Component<Props, State> {
 
         // Otherwise let's focus the first tabbable item and expedite input from the user.
         if (this.content) {
-          const tabbableItems = tabbable(this.content);
+          const tabbableItems = tabbable(this.content, {
+            displayCheck: 'legacy-full',
+          });
```

Comment:
Why is this the only instance where options are added to the `tabbable` call? Is it special for some reason? Should the other calls to `tabbable` have it too?

Reply:
The rest of the cases do not cause problems with the tests and smoothly support the transition to the new version. In this file apparently a different case, after the upgrade the tests did not behave correctly, so I had to apply `legacy-full`.

Reply:
Let's open a follow-up issue to see if we can refactor and remove.